pkg/controller/installation/installation_controller.go

```diff
@@ -86,20 +96,30 @@ type ReconcileInstallation struct {
 	coreClient        *kubernetes.Clientset
 	restConfig        *rest.Config
 	productsToInstall []string
+	context           context.Context
+	cancel            context.CancelFunc
 }
 
 // Reconcile reads that state of the cluster for a Installation object and makes changes based on the state read
 // and what is in the Installation.Spec
 func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.Result, error) {
```

Comment:
for instance it would normally passed here ```(ctx, request)```

Reply:
I don't call this method, it's called from the operator-sdk code.